### PR TITLE
update gradle major version

### PR DESCRIPTION
**Pull request by e7mac**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/BaseMessageDialog.java [hunk 1]
```diff
@@ -105,7 +106,7 @@ protected BaseMessageDialog(Activity activity, boolean fullscreen, BaseMessageOp
     dialogView.setLayoutParams(layoutParams);
 
     RelativeLayout view = createContainerView(activity, fullscreen);
-    view.setId(108);
```
Comment (hrishileanplum) [inline AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/BaseMessageDialog.java]: Wow

File: AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/BaseMessageDialog.java [hunk 2]
```diff
@@ -209,7 +210,7 @@ public void run() {
 
   private CloseButton createCloseButton(Activity context, boolean fullscreen, View parent) {
     CloseButton closeButton = new CloseButton(context);
-    closeButton.setId(103);
```
Comment (hrishileanplum) [inline AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/BaseMessageDialog.java]: Thank you for these changes :)

File: AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java [hunk 3]
```diff
@@ -299,6 +300,7 @@ static void setModifiedBigPictureStyle(Notification.Builder notificationBuilder,
    * @param messageText String with text for push notification.
    * @return Notification.BigPictureStyle or null.
    */
+  @TargetApi(16)
```
Comment (hrishileanplum) [inline AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java]: Should these be changed from 16-26 as well?

File: AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java [hunk 4]
```diff
@@ -299,6 +300,7 @@ static void setModifiedBigPictureStyle(Notification.Builder notificationBuilder,
...
   static Notification.BigPictureStyle getBigPictureStyle(Bundle message, Bitmap bigPicture,
       String title, final String messageText) {
     if (Build.VERSION.SDK_INT < 16 || message == null || bigPicture == null) {
```
Comment (hrishileanplum) [inline AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java]: Yea, i see mix of api target 16 or 26. Any ideas why?

File: AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java (hunk 3 above)
Comment (e7mac) [reply AndroidSDKPush/src/main/java/com/leanplum/LeanplumNotificationHelper.java]: Yeah. I'm putting the lowest needed TargetApi numbers in here. 
Here's why
1. The gradle compilation process checks to ensure that we aren't using any android calls that are below the minimum supported version (14 in this case)
2. However, in certain methods, if we are handling the different API calls ourselves (through branching etc), then we can let the compiler know that and ask it for a higher API check.
3. In this case, I'm setting the TargetApi wrt the branching logic in the function, 16 in most cases, but 26 for notification channels.

**Pull request merged**